src/nimble.nim

```diff
@@ -410,9 +411,9 @@ proc installFromDir(dir: string, requestedVer: VersionRange, options: Options,
   var filesInstalled = initHashSet[string]()
   iterInstallFiles(realDir, pkgInfo, options,
     proc (file: string) =
-      createDir(changeRoot(realDir, pkgDestDir, file.splitFile.dir))
```

Comment:
This is a nimpretty bug (@narimiran), however not blocking the PR and ofc not your fault.